src/sync/Synchroniser.js

```diff
@@ -60,6 +58,8 @@ export class Synchroniser {
     this.extraHeaders = {
       'msupply-site-uuid': DeviceInfo.getUniqueId(),
       'mobile-version': mobileVersion,
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
```

Comment:
Needed to add this actually because of the node bump: I suppose `fetch` impl changed and needed these to send `POST`